FluidNC/src/Motors/TrinamicSpiDriver.cpp

```diff
@@ -34,7 +34,8 @@ namespace MotorDrivers {
         }
 
         if (_driver_part_number == 2130) {
-            tmcstepper = new TMC2130Stepper(cs_id, _r_sense, -1);  // TODO hardwired to non daisy chain index
+            //log_info("ID: " << cs_id << " index:" << _spi_index);
+            tmcstepper = new TMC2130Stepper(cs_id, _r_sense, _spi_index);  // TODO hardwired to non daisy chain index
```

Comment:
I guess the TODO can go

Reply:
not working but there is a change, Z motor passed the test :
![image](https://user-images.githubusercontent.com/12842861/135692022-49a79294-9d8e-4bd6-838c-799e7ee94767.png)